neqo-transport/src/connection/mod.rs

```diff
@@ -746,7 +750,11 @@ impl Connection {
         if !init_token.is_empty() {
             self.address_validation = AddressValidationInfo::NewToken(init_token.to_vec());
         }
-        self.paths.primary().borrow_mut().rtt_mut().set_initial(rtt);
+        if let Some(path) = self.paths.primary() {
+            path.borrow_mut().rtt_mut().set_initial(rtt);
+        } else {
+            return Err(Error::InternalError);
+        }
```

Comment:
```suggestion
        self.paths.primary().ok_or(Error::InternalError)?.borrow_mut().rtt_mut().set_initial(rtt);
```

`rustfmt` might have something to say about that one though.